File: app/src/main/java/com/example/tugasbesarkotlin3/Models/Menu.java
package com.example.tugasbesarkotlin3.Models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;


public class Menu {
	@SerializedName("image")
	@Expose
	private String image;
	@SerializedName("product")
	@Expose
	private String product;
	@SerializedName("price")
	@Expose
	private String price;

	public String getImage() {
		return image;
	}

	public String getProduct() {
		return product;
	}

	public String getPrice() {
		return price;
	}
}

File: app/src/main/java/com/example/tugasbesarkotlin3/Paging/Menu.kt
package com.example.tugasbesarkotlin3.Paging

import com.google.gson.annotations.SerializedName

data class Menu(
    @field:SerializedName("id")
    val id: Int? = null,

    @field:SerializedName("product")
    val product: String? = null,

    @field:SerializedName("image")
    val image: String? = null,

    @field:SerializedName("price")
    val price: String? = null
)